docs_src/microservices/device/details/AutoEvents.md

```diff
@@ -3,5 +3,25 @@ title: Device Service - Auto Events
 ---
 
 # Device Service - Auto Events
+## Concept
+![image](Auto_Events.png)
+In the Operational Technology (OT) working field, there's a need to check/menitor the status of the actual device/sensor, for example, when the temperature reaches 30°C, the Air Conditioner will turn on itself. However, it is time-consuming to check sensor's status manually so edgex provides this functionality called <code>AutoEvents</code> to check their statuses automatically and periodically.
 
-coming soon
\ No newline at end of file
+From the diagram above, <code>Autoevents</code> is used to define how often events/readings are collected to be sent to core data from the device service. This is an optional feature when creating a deviceso each device may or may not have multiple autoevents associated with it. An AutoEvent has the following fields:
```

Comment:
The diagram doesn't work for me. Why is Core Metadata sending reading to Core Data.

Device service should "Get" reading(s) from device and send event/reading to MessageBus when App service/Core Data will receive them.

Reply:
thanks Lenny, updated